package.json

```diff
@@ -16,11 +16,11 @@
     "eslint-config-airbnb-base": "^11.1.1",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-mocha": "^4.9.0",
-    "html-webpack-plugin": "^2.28.0",
+    "html-webpack-plugin": "webpack-contrib/html-webpack-plugin",
     "mocha": "^3.2.0",
-    "uglify-js": "^2.8.22",
-    "uglifyjs-webpack-plugin": "^0.4.2",
-    "webpack": "^2.4.1"
+    "rimraf": "^2.6.2",
+    "webpack": "^4.0.1",
+    "webpack-cli": "^2.0.9"
```

Comment:
Where are we using this?

Reply:
webpack 4 requires webpack-cli to build now